pkg/apis/agones/v1/gameserver.go

```diff
@@ -135,16 +135,20 @@ type GameServerSpec struct {
 	Health Health `json:"health,omitempty"`
 	// Scheduling strategy. Defaults to "Packed"
 	Scheduling apis.SchedulingStrategy `json:"scheduling,omitempty"`
-	// Logging specifies log levels for Agones system containers
-	Logging Logging `json:"logging,omitempty"`
+	// SdkServer specifies parameters for the Agones SDK Server sidecar container
+	SdkServer SdkServer `json:"sdkServer,omitempty"`
 	// Template describes the Pod that will be created for the GameServer
 	Template corev1.PodTemplateSpec `json:"template"`
 }
 
-// Logging specifies log levels for Agones system containers
-type Logging struct {
-	// SdkServer is a log level for SDK server (sidecar) logs. Defaults to "Info"
-	SdkServer string `json:"sdkServer,omitempty"`
+// SdkServer specifies parameters for the Agones SDK Server sidecar container
+type SdkServer struct {
+	// Logging is a log level for SDK server (sidecar) logs. Defaults to "Info"
```

Comment:
```suggestion
	// Logging is a log level for SDK server (sidecar) logs. Defaults to "Info". Can also be "Error" or "Warning"
```

Reply:
I actually think that with a type defined for this we don't need to enumerate all possible values here (but saying the default is still useful).